datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/util/DurationUnitConverter.java

```diff
@@ -0,0 +1,76 @@
+package com.fasterxml.jackson.datatype.jsr310.util;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.time.temporal.TemporalUnit;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import static com.fasterxml.jackson.datatype.jsr310.util.DurationUnitConverter.DurationSerialization.deserializer;
+
+
```

Comment:
Could you add `@since 2.12` here?

Reply:
Done, I added documentation to the class too.